docs/content/docs/deployment/overview.md

```diff
@@ -321,7 +321,7 @@ Supported Environment:
 
 #### Ververica Platform
 
-[Website](https://www.ververica.com/platform-overview)
+[Website]https://www.ververica.com/platform)
```

Comment:
Why `(` is removed.

Reply:
Good catch. By mistake I guess, I will fix